internal/cli/atlas/setup/setup_cmd.go

```diff
@@ -605,24 +626,19 @@ func Builder() *cobra.Command {
 	cmd.Flags().BoolVar(&opts.register.IsGov, "gov", false, "Register with Atlas for Government.")
 	cmd.Flags().BoolVar(&opts.register.NoBrowser, "noBrowser", false, "Don't try to open a browser session.")
 	// Setup related
+	opts.SetupAtlasFlags(cmd)
+	opts.SetupFlowFlags(cmd)
+
+	// TODO: Add support to MDB version and Connect with to setup command
+	var str string
+	cmd.Flags().StringVar(&str, flag.MDBVersion, "", usage.MDBVersion)
+	_ = cmd.Flags().MarkHidden(flag.MDBVersion)
+	cmd.Flags().StringVar(&str, flag.ConnectWith, "", usage.ConnectWith)
+	_ = cmd.Flags().MarkHidden(flag.ConnectWith)
```

Comment:
for now, what about removing these flags in deployments setup where we do `workflows.RemoveFlagsAndArgs(...`